### PR TITLE
[DB-6768]Changed the default delimiter according to file format csv (',') and text ('\t')

### PR DESCRIPTION
**Pull request by priyanshi-yb**
Fixes - [DB-6768](https://yugabyte.atlassian.net/browse/DB-6768)

### CODE REVIEW

File: yb-voyager/cmd/importDataFileCommand.go [hunk 1]
```diff
@@ -367,8 +381,8 @@ func init() {
 	importDataFileCmd.Flags().StringVar(&fileFormat, "format", "csv",
 		fmt.Sprintf("supported data file types: %v", supportedFileFormats))
 
-	importDataFileCmd.Flags().StringVar(&delimiter, "delimiter", ",",
-		"character used as delimiter in rows of the table(s)")
+	importDataFileCmd.Flags().StringVar(&delimiter, "delimiter", "",
+		`character used as delimiter in rows of the table(s)(default for csv: ',' andgit  for text: '\t')`)
```
Comment (amit-yb) [inline yb-voyager/cmd/importDataFileCommand.go:385]: `git   `???

Reduce the number of words to convey the information:
(DEFAULT: comma for CSV and tab for TEXT format)

File: yb-voyager/cmd/importDataFileCommand.go [hunk 2]
```diff
@@ -326,6 +327,19 @@ func setDefaultForNullString() {
 	}
 }
 
+func setDefaultForDelimiter() {
+	if delimiter == "" {
```
Comment (amit-yb) [inline yb-voyager/cmd/importDataFileCommand.go:331]: Nit: Not much of a problem for this specific function. But in idiomatic go, we try to reduce indentation by inverting the checks and returning early e.g. this function can be implemented as:
```
func setDefaultForDelimiter() {
  if delimiter != "" {
    return
  }
  switch fileFormat {
    ...
  }
}
```
Think as if you pay Rs 10 for the first level of indentation, Rs 100 for the second level, 1000 for the next, and so on!! Now the objective is to save as much money as you can ;-)

File: migtests/tests/import-file/run-import-file-test [hunk 3]
```diff
@@ -84,6 +84,15 @@ main() {
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter ',' \
 			--file-table-map "t1_newline.csv:t1_newline"
 
+	# Test for text file for default delimiter
+	step "Import data file: test_default_delimiter.csv -> test_default_delimiter"
+	import_data_file --data-dir ${TEST_DIR} --format text  \
+			--file-table-map "test_default_delimiter.txt:test_default_delimiter"
```
Comment (amit-yb) [inline migtests/tests/import-file/run-import-file-test:90]: Thanks for adding the tests 👍 !!

**Review by amit-yb — APPROVED** [review]
LGTM with couple of minor comments.

File: yb-voyager/cmd/importDataFileCommand.go (hunk 2 above)
Comment (priyanshi-yb) [reply yb-voyager/cmd/importDataFileCommand.go:331]: Yes, got it. Thanks for the example :)

**Pull request merged**